ibm/mas_devops/roles/ocp_setup_mas_deps/defaults/main.yml

```diff
@@ -17,6 +17,11 @@ prometheus_storage_size: "{{ lookup('env', 'PROMETHEUS_STORAGE_SIZE') | default(
 prometheus_alertmgr_storage_class: "{{ lookup('env', 'PROMETHEUS_ALERTMGR_STORAGE_CLASS') }}"
 prometheus_alertmgr_storage_size: "{{ lookup('env', 'PROMETHEUS_ALERTMGR_STORAGE_SIZE') | default('20Gi', true) }}"
 
+# Settings to update openshift user workload monitoring to define a specific storage class and size for Prometheus logs
```

Comment:
Can you update the documentation (in the README) with the details for the new variables and env vars that have been introduced please?

Reply:
@durera README is updated